### ISSUE TITLE
Do not transfer selected (not filtering correctly)

### ISSUE THREAD

**Issue by jcruz0886** [finndev/PokeBuddy#676]
**Do not transfer selected** filter is not working correctly, loop will start transferring Pokemon from the list(Do not transfer selected).

**Comment by curtiola** [comment]
Yeah it's a known issue, hope it will be fixed soon, and perhaps the addition of being able to choose an amount of each one to keep :D

**Comment by perfidiousdan** [comment]
Yeah it just transfering everything lile a douch bot. Bad robot

**Comment by curtiola** [comment]
Bad gir

**Comment by joejoejoey04** [comment]
If it could read IV's and keep pokemon with the potential to be in the higher percentile would be nice
